os/kernel/binary_manager/Kconfig

```diff
@@ -45,6 +45,13 @@ config BINMGR_UPDATE
 	---help---
 		Enables Binary Manager Update APIs.
 
+config BINMGR_ALLOW_SAME_VERSION_UPDATE
+	bool "Allow Same Version Binary Update"
+	default y
+	depends on BINMGR_UPDATE
+	---help---
+		Allow same version binary update.
```

Comment:
Without this config, only higher version can be updated.
How about elaborating this in the help message also?